packages/react/src/PageMenu/PageMenu.tsx

```diff
@@ -0,0 +1,74 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import clsx from 'clsx'
+import {
+  AnchorHTMLAttributes,
+  ButtonHTMLAttributes,
+  ForwardedRef,
+  forwardRef,
+  ForwardRefExoticComponent,
+  HTMLAttributes,
+  PropsWithChildren,
+  RefAttributes,
+} from 'react'
+import { Icon } from '../Icon'
+
+type PageMenuProps = PropsWithChildren<HTMLAttributes<HTMLElement>>
+
+interface PageMenuComponent extends ForwardRefExoticComponent<PageMenuProps & RefAttributes<HTMLElement>> {
+  Link: typeof PageMenuLink
+  Button: typeof PageMenuButton
+}
+
+export const PageMenu = forwardRef(
+  ({ children, className, ...restProps }: PageMenuProps, ref: ForwardedRef<HTMLElement>) => {
+    return (
+      <nav {...restProps} className={clsx('amsterdam-page-menu', className)} ref={ref}>
```

Comment:
For the footer the list shouldn't be wrapped in a `nav`, I think. Maybe PageMenu should just be `ul` -> `li` -> `link` / `button`, and the `nav` should be wrapped around it when composing the Header?

Reply:
Good point, I'll take it out.

Reply:
I agree. See this quote from the spec: “In particular, it is common for footers to have a list of links to various key parts of a site, but the footer element is more appropriate in such cases, and no nav element is necessary for those links.” ([source](http://html5doctor.com/nav-element/))

Please add documentation that implementers must wrap a Page Menu in the page header in a `nav` element, and they must not do so in the page footer.